composer.json

```diff
@@ -95,8 +95,8 @@
         "ci:tests": [
             "@ci:tests:unit"
         ],
-        "ci:tests:sof": "@php \"./vendor/bin/phpunit\" --stop-on-failure",
-        "ci:tests:unit": "@php \"./vendor/bin/phpunit\"",
+        "ci:tests:sof": "@php \"./vendor/bin/phpunit\" --stop-on-failure --do-not-cache-result",
+        "ci:tests:unit": "@php \"./vendor/bin/phpunit\" --do-not-cache-result",
```

Comment:
Yep, caching often causes problems if not absolutely perfectly thought out and implemented.  In our case, it is unlikely to save any significant CPU time, though we should always be mindful of the CO<sub>2</sub> footprint.